torchgeo/datasets/mmearth.py

```diff
@@ -0,0 +1,620 @@
+            'curr_temperature_2m_min',  # current month min temp
+            'curr_temperature_2m_max',  # current month max temp
+            'curr_total_precipitation_sum',  # current month total precip
+            '0_temperature_2m_mean',  # year avg temp
+            '1_temperature_2m_min_min',  # year min temp
+            '2_temperature_2m_max_max',  # year max temp
+            '3_total_precipitation_sum_sum',  # year total precip
+        ],
+        'dynamic_world': ['label'],
+        'canopy_height_eth': ['height', 'std'],
+        'lat': ['sin', 'cos'],
+        'lon': ['sin', 'cos'],
+        'biome': ['biome'],
+        'eco_region': ['eco_region'],
+        'month': ['sin_month', 'cos_month'],
+        'esa_worldcover': ['Map'],
+    }
+
+    # See https://github.com/vishalned/MMEarth-train/blob/8d6114e8e3ccb5ca5d98858e742dac24350b64fd/MODALITIES.py#L36
+    no_data_vals: ClassVar[dict[str, int | float]] = {
+        'sentinel2': 0,
+        'sentinel2_cloudmask': 65535,
+        'sentinel2_cloudprod': 65535,
+        'sentinel2_scl': 255,
+        'sentinel1_asc': float('-inf'),
+        'sentinel1_desc': float('-inf'),
+        'aster': float('-inf'),
+        'canopy_height_eth': 255,
+        'dynamic_world': 0,
+        'esa_worldcover': 255,
+        'lat': float('-inf'),
+        'lon': float('-inf'),
+        'month': float('-inf'),
+        'era5': float('inf'),
+        'biome': 255,
+        'eco_region': 65535,
+    }
+
+    norm_modes: ClassVar[list[str]] = ['z-score', 'min-max']
+
+    modality_category_name: ClassVar[dict[str, str]] = {
+        'sentinel1_asc': 'image_',
+        'sentinel1_desc': 'image_',
+        'sentinel2': 'image_',
+        'sentinel2_cloudmask': 'mask_',
+        'sentinel2_cloudprod': 'mask_',
+        'sentinel2_scl': 'mask_',
+        'aster': 'image_',
+        'era5': '',
+        'canopy_height_eth': 'image_',
+        'dynamic_world': 'mask_',
+        'esa_worldcover': 'mask_',
+    }
+
+    def __init__(
+        self,
+        root: Path = 'data',
+        subset: str = 'MMEarth',
+        modalities: Sequence[str] = all_modalities,
+        modality_bands: dict[str, list[str]] | None = None,
+        normalization_mode: str = 'z-score',
+        transforms: Callable[[dict[str, Tensor]], dict[str, Tensor]] | None = None,
+    ) -> None:
+        """Initialize the MMEarth dataset.
+
+        Args:
+            root: root directory where dataset can be found
+            subset: one of "MMEarth", "MMEarth64", or "MMEarth100k"
+            modalities: list of modalities to load
+            modality_bands: dictionary of modality bands, see `all_modality_bands`
+            normalization_mode: one of "z-score" or "min-max"
+            transforms: a function/transform that takes input sample dictionary
+                and returns a transformed version
+
+        Raises:
+            AssertionError: if ``normalization_mode``, ``subset``, or ``split`` is invalid
```

Comment:
```suggestion
            AssertionError: if *normalization_mode*, *subset*, or *split* is invalid
```
The Python and Sphinx docs use italics to reference parameters, trying to match them.